### PR TITLE
FF86 Release notes for AVIF addition

### PR DESCRIPTION
**Pull request by hamishwillee**
Adds release notes about 
- addition of basic AVIF support. 
- Update of HTTP header ACCEPT for HTML files following AVIF support.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/86/index.html [hunk 1]
```diff
@@ -53,6 +57,17 @@ <h2 id="Changes_for_add-on_developers">Changes for add-on developers</h2>
 
 <h4 id="Removals_9">Removals</h4>
 
+<h3 id="Other">Other</h3>
+
+<ul>
+  <li>Basic support for <a href="/en-US/docs/Web/Media/Formats/Image_types#avif">AVIF</a> images is now enabled by default ({{bug(1682995)}}).
+    <ul>
+      <li><a href="/en-US/docs/Web/Media/Formats/Image_types#avif">AVIF</a> is a new image format with excellent compression and no patent restrictions developed by the Alliance for Open Media.</li>
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/firefox/releases/86/index.html:65]: maybe provide a link to the Alliance for Open Media? Sounds interesting, and people would want to know more about it.

File: files/en-us/mozilla/firefox/releases/86/index.html [hunk 2]
```diff
@@ -31,6 +31,10 @@ <h4 id="Removals_4">Removals</h4>
 
 <h3 id="HTTP">HTTP</h3>
 
+<ul>
+  <li>The default HTTP {{HTTPHeader("ACCEPT")}} header for <em>images</em> changed to: <code>image/avif,image/webp,*/*</code> (following addition of support for the <a href="/en-US/docs/Web/Media/Formats/Image_types#avif">AVIF</a> image format).</li>
```
Comment (chrisdavidmills) [inline files/en-us/mozilla/firefox/releases/86/index.html:35]: bug number?

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
cheers @hamishwillee ; couple of small requests for change, but nothing major

**Comment by hamishwillee** [comment]
Thanks @chrisdavidmills , suggestions integrated. Read for rechecking.

**Review by chrisdavidmills — APPROVED** [review]
This now looks good; thanks @hamishwillee !

**Pull request merged**